browser/ui/webui/brave_education/education_page_ui.h

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_EDUCATION_EDUCATION_PAGE_UI_H_
+#define BRAVE_BROWSER_UI_WEBUI_BRAVE_EDUCATION_EDUCATION_PAGE_UI_H_
+
+#include <memory>
+
+#include "brave/components/constants/webui_url_constants.h"
+#include "brave/ui/webui/resources/js/brave_browser_command/brave_browser_command.mojom.h"
+#include "content/public/browser/webui_config.h"
+#include "content/public/common/url_constants.h"
+#include "mojo/public/cpp/bindings/pending_receiver.h"
+#include "mojo/public/cpp/bindings/pending_remote.h"
+#include "mojo/public/cpp/bindings/receiver.h"
+#include "ui/webui/mojo_web_ui_controller.h"
+#include "url/gurl.h"
+
+namespace content {
+class WebUI;
+}
+
+namespace brave_education {
+
+// The Web UI controller for the Brave product education page, which displays
+// production education website content in an iframe.
+class EducationPageUI
```

Comment:
This should have `BraveEducationPageUIConfig`. This method of creating webui is deprecated.

Reply:
errr... I guess it's named `GettingStartedUIConfig`? So please change the name to match